ivy/container/base.py

```diff
@@ -684,8 +684,20 @@ def multi_map(
                     )
                     if ret:
                         return_dict[key] = ret
-                elif isinstance(value0, (list, tuple)) and map_nests:
-                    ret = ivy.nested_multi_map(lambda x, _: func(x, None), values)
+                elif any(isinstance(x, (list, tuple)) for x in values) and map_nests:
+                    to_ivy = False
+                    for value in values:
```

Comment:
could you add comments explaining why this is needed? I would have thought the function being mapped itself converts things to `ivy`? Why do we need to hard code this here you think?

Reply:
multi_map handles leaves that contain lists or tuples by delegating the work to the nested_multi_map function which by default always returns ivy arrays regardless of the function being mapped. However, for operator functions, if the leaves only contain numbers and not ivy arrays, nested_multi_map should be instructed to return number values and not ivy arrays through the to_ivy variable.